src/Offer/ReadModel/Metadata/OfferMetadataRepository.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\ReadModel\Metadata;
+
+use CultuurNet\UDB3\EntityNotFoundException;
+use CultuurNet\UDB3\Label\ReadModels\Doctrine\AbstractDBALRepository;
+use Doctrine\DBAL\Connection;
+use ValueObjects\StringLiteral\StringLiteral;
+
+class OfferMetadataRepository extends AbstractDBALRepository
+{
+    public const TABLE = 'offer_metadata';
+
+    public function __construct(Connection $connection)
+    {
+        parent::__construct($connection, new StringLiteral(self::TABLE));
+    }
+
+    public function get(string $offerId)
```

Comment:
This only seems to be used inside the tests?

Reply:
Yes, we use it in the follow-up PR to enrich the jsonld